src/transaction/log_storage.hpp

```diff
@@ -168,14 +168,18 @@ struct log_header
 
   INT64 ha_promotion_time;
   INT64 db_restore_time;
+
   bool mark_will_del;
   bool does_block_need_vacuum;
   bool was_active_log_reset;
+  /* Here exists 5 bytes */
+
+  INT64 vol_creation_time; /* Volume creation time */
```

Comment:
NIT:
변수명으로 유추할 수 있는 불필요한 주석은 남길 필요가 없습니다. 기존 코드에도 많이 있긴 합니다. 정보 전달이 필요한 의미있는 주석만 남기면 되겠습니다.

```suggestion
  INT64 vol_creation_time;
```

Reply:
제거 하였습니다.